examples/open_loop_control_fetch/user_config.yaml.example

```diff
@@ -15,6 +15,8 @@ SPEAR:
   WINDOW_RESOLUTION_Y: 512
 
 SIMULATION_CONTROLLER:
+  MAX_SUBSTEP_DELTA_TIME: 0.001
```

Comment:
move after CUSTOM_UNREAL_CONSOLE_COMMANDS so it's in the same order as in default_config